velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -229,9 +229,10 @@ TEST_F(DateTimeFunctionsTest, lastDay) {
 
 TEST_F(DateTimeFunctionsTest, dateAdd) {
   const auto dateAdd = [&](std::optional<int32_t> date,
-                           std::optional<int32_t> value) {
+                           std::optional<int32_t> value,
+                           std::string expr = "date_add(c0, c1)") {
     return evaluateOnce<int32_t, int32_t>(
-        "date_add(c0, c1)", {date, value}, {DATE(), INTEGER()});
+        expr, {date, value}, {DATE(), INTEGER()});
```

Comment:
Would it be clearer to use a separate lambda for tinyint and smallint tests? Or maybe separate the tests for different types into different unit tests.

Reply:
Thanks, I split the test cases.